css-align-3/content-distribution/place-content-shorthand-001.html

```diff
@@ -8,8 +8,13 @@
 <script src="/css-align-3/resources/alignment-parsing-utils.js"></script>
 <div id="log"></div>
 <script>
+    function checkValue(alignValue)
```

Comment:
I don't think we need this oneliner method, that is only called in 1 place.
I'll just write it inside the `test()` call.
IMHO, this applies to all oneliner methods you have that are called just in 1 place.

Also you were calling this with 2 arguments, but only having 1. Strange.